container/kvm/initialisation.go

```diff
@@ -82,6 +74,24 @@ func ensureDependencies() error {
 	return nil
 }
 
+func getRequiredPackages(a string) []string {
+	var requiredPackages = []string{
+		// `qemu-kvm` must be installed before `libvirt-bin` on trusty. It appears
+		// that upstart doesn't reload libvirtd if installed after, and we see
+		// errors related to `qemu-kvm` not being installed.
+		"qemu-kvm",
+		"genisoimage",
+		"libvirt-bin",
+		"qemu-utils",
+	}
+	if a == arch.ARM64 {
+		// ARM64 doesn't support legacy BIOS so it requires Extensible Firmware
+		// Interface.
+		requiredPackages = append(requiredPackages, "qemu-efi")
```

Comment:
The issue with order is an upstart issue, it apparently doesn't reload libvirt after installing qemu-kvm, but no big deal. Reordered to put all the qemu packages first. If on ARM64 the efi package is prepended to the slice. It has no dependencies so it should be fine up front.